### ISSUE TITLE
Assertion error in props wrapper

### ISSUE THREAD

**Issue by gabrielStanovsky** [vered1986/OKR#12]
@kleinay, can you please post an example sentence which gave you the following error - 

```
Traceback (most recent call last):
  File "src/baseline_automatic_pipeline_system/okr_for_mds.py", line 62, in <module>
    okr_info = auto_pipeline_okr_info(tweets_strings)
  File "/home/ir/kleinay/OKR/src/baseline_automatic_pipeline_system/parse_okr_info.py", line 240, in auto_pipeline_okr_info
    parsed_sentences = parse_single_sentences(sentences)
  File "/home/ir/kleinay/OKR/src/baseline_automatic_pipeline_system/parse_okr_info.py", line 49, in parse_single_sentences
    pw.parse(sent)
  File "src/baseline_system/parsers/props_wrapper.py", line 100, in parse
    self.parse_okr()
  File "src/baseline_system/parsers/props_wrapper.py", line 109, in parse_okr
    self.parse_predicate(pred)
  File "src/baseline_system/parsers/props_wrapper.py", line 232, in parse_predicate
    dep_tree = self.get_dep_node(predicate_node)
  File "src/baseline_system/parsers/props_wrapper.py", line 176, in get_dep_node
    self.dep_tree
AssertionError: Problems matching 14; nodes matched were:[<props.dependency_tree.tree.DepTree object at 0x115a61d0>, <props.dependency_tree.tree.DepTree object at 0x115a6210>]; dep tree: [<props.dependency_tree.tree.DepTree object at 0x11577f50>, <props.dependency_tree.tree.DepTree object at 0x11577e10>, <props.dependency_tree.tree.DepTree object at 0x11577ed0>, <props.dependency_tree.tree.DepTree object at 0x11577e90>, <props.dependency_tree.tree.DepTree object at 0x11577e50>, <props.dependency_tree.tree.DepTree object at 0x11577fd0>, <props.dependency_tree.tree.DepTree object at 0x11577f10>, <props.dependency_tree.tree.DepTree object at 0x115a6050>, <props.dependency_tree.tree.DepTree object at 0x115a6090>, <props.dependency_tree.tree.DepTree object at 0x115a60d0>, <props.dependency_tree.tree.DepTree object at 0x115a6110>, <props.dependency_tree.tree.DepTree object at 0x115a6150>, <props.dependency_tree.tree.DepTree object at 0x115a6190>, <props.dependency_tree.tree.DepTree object at 0x115a61d0>, <props.dependency_tree.tree.DepTree object at 0x115a6210>, <props.dependency_tree.tree.DepTree object at 0x115a6250>, <props.dependency_tree.tree.DepTree object at 0x115a62d0>, <props.dependency_tree.tree.DepTree object at 0x115a6290>, <props.dependency_tree.tree.DepTree object at 0x115a6310>, <props.dependency_tree.tree.DepTree object at 0x115a6350>, <props.dependency_tree.tree.DepTree object at 0x115a6390>, <props.dependency_tree.tree.DepTree object at 0x115a63d0>]
```

Thanks!

**Comment by gabrielStanovsky** [comment]
Even better, the input file + command which gives this output.

**Comment by kleinay** [comment]
It fails when running the pipeline on real tweet file. 
Run the following command from the OKR base directory:
_python src/baseline_automatic_pipeline_system/okr_for_mds.py --out=tmp --tweets=examples/tweets/boy_scouts_tweets --meta=examples/tweets/tweets_metadata_

An example for sentence that raises this error (tweet 259060960092893184 from boy_scouts scenerio):
"Horrified by national news headlines this evening that  the Boy Scouts USA covered up allegations of child abuse for years."

**Comment by gabrielStanovsky** [comment]
I addressed this in #15 
@kleinay, please give the new commit a quick look, and I'll merge this.
BTW, I think you're gonna have a little bit of merge issues when you pull the new changes, since you also changed the PropS wrapper file in your fork. We can do it together, if you want.

Thanks

**Comment by gabrielStanovsky** [comment]
Fixed, closing issue.
